doc/examples/big_data_ingestion.py

```diff
@@ -0,0 +1,146 @@
+# flake8: noqa: E501
+"""
+Large-scale shuffle ingestion pipeline
```

Comment:
I'm wondering if this should be something like "Large-scale machine learning ingestion pipeline" or "Large-scale distributed training ingestion pipeline", since that's what this pipeline is truly providing (shuffling is just one feature of the ingestion pipeline) and this might do a better job of grabbing the attention of a user that's looking for such a pipeline, since "shuffle ingestion pipeline" might not connote ML model training for some of our users. If we're wanting to explicitly call out the shuffle since it's a big feature, then we could do something like "Large-scale per-epoch shuffling ingestion pipeline for distributed training". cc @ericl for thoughts here